src/resources/users/tests/user-router.spec.js

```diff
@@ -0,0 +1,114 @@
+const request = require('supertest');
+const server = require('../../../../index');
+// const db = require('../data/dbConfig');
+
+// beforeEach(() => {
+//   return db('users_table').truncate()
+// })
+
+describe('user', () => {
+  
+  describe('[POST] / endpoint', () => {
+
+    const jayneData = {"first_name": "Jayne", "last_name": "Carmichael Norrie", "email": "jayne@musicisourforte.co.uk", "password": "chico", "user_role_id": 2}
+
```

Comment:
Hey Jayne, awesome work with the tests, all are passing! Let's try to delete all commented out code and to use single quotes in the next Pull Requests to have uniform code. I will merge this PR now.